.github/workflows/ci.yml

```diff
@@ -0,0 +1,89 @@
+name: CI
+on:
+  push:
+    tags:
+      - '*'
+
+jobs:
+  build:
+    name: Build system image
+    runs-on: self-hosted
+    steps:
+      - name: Checkout meta-dts-ce repo
+        uses: actions/checkout@v2
+        with:
+          path: "meta-dts-ce"
+      - name: Build DTS CE image
+        shell: bash
+        run: |
+          kas-container shell meta-dts-ce/kas.yml -c "bitbake -c cleanall dts-base-image-ce"
+          kas-container shell meta-dts-ce/kas.yml -c "bitbake dts-base-image-ce"
+  deploy-cache:
+    name: Deploy cache on cache.dasharo.com
+    if: always()
+    needs: build
+    runs-on: self-hosted
+    steps:
+      - name: Prepare SSH key
+        shell: bash
+        env:
+          SSH_KEY: ${{secrets.SSH_CACHE}}
```

Comment:
We should name the secret differently as it is shared beetwen cache and boot

Reply:
done

Reply:
Please, add also link to the change so I could verify if there is something I can agree on

Reply:
https://github.com/Dasharo/meta-dts-ce/pull/5/commits/dc8e0518ceebb7499a4bfce675eb9e60fb271c42#diff-b803fcb7f17ed9235f1e5cb1fcd2f5d3b2838429d4368ae4c57ce4436577f03fR29